### ISSUE TITLE
How can I get the training loss ?

### ISSUE THREAD

**Issue by zhanghaok** [TeamHG-Memex/sklearn-crfsuite#73]
How can I get the training loss ?
